x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/types.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface Meta {
+  from: number;
+  size: number;
+  total: number;
+}
+
+export interface EngineListDetails {
```

Comment:
We'll want to replace this with the types from the `common` directory now that they exist. But that can be done in a follow-up PR if need be.

Reply:
Yes. Will do in next PR.